src/adapters/CKEditorAdapter.ts

```diff
@@ -14,51 +14,28 @@
  * limitations under the License.
  */
 
-import { Match, MatchWithReplacement } from '@acrolinx/sidebar-interface';
-import { AbstractRichtextEditorAdapter } from './AbstractRichtextEditorAdapter';
-import { HasEditorID, ContentExtractionResult } from './AdapterInterface';
```

Comment:
There is a change in liniting here - is this new Linting correct? Is linting handled so that it is universal in the repo?

Reply:
@JustinHorn I have added prettier.

Currently prettier gives over 1000 errors, to fix all of them is a ticket in itself I think. I will create a new ticket specifically to reformat code.

```
1386 problems (0 errors, 1386 warnings)
  0 errors and 1386 warnings potentially fixable with the `--fix` option.
```

The lines you mentioned in the comment are deleted.

Reply:
okay 👍🏻